filebeat/Jenkinsfile.yml

```diff
@@ -13,6 +13,11 @@ when:
     tags: true                 ## for all the tags
 platform: "linux && ubuntu-18" ## default label for all the stages
 stages:
+    Lint:
```

Comment:
Nit. This stage is upercased in some files, and lowercased in others.

Reply:
yep I saw it on the GitHub checks